src/index.js

```diff
@@ -0,0 +1,76 @@
+/**
+*
+* @licstart  The following is the entire license notice for the JavaScript code in this file.
+*
+* RESTful API for Melinda
+*
+* Copyright (C) 2018-2019 University Of Helsinki (The National Library Of Finland)
+*
+* This file is part of melinda-rest-api-http
+*
+* melinda-rest-api-http program is free software: you can redistribute it and/or modify
+* it under the terms of the GNU Affero General Public License as
+* published by the Free Software Foundation, either version 3 of the
+* License, or (at your option) any later version.
+*
+* melinda-rest-api-http is distributed in the hope that it will be useful,
+* but WITHOUT ANY WARRANTY; without even the implied warranty of
+* MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+* GNU Affero General Public License for more details.
+*
+* You should have received a copy of the GNU Affero General Public License
+* along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*
+* @licend  The above is the entire license notice
+* for the JavaScript code in this file.
+*
+*/
+
+import {Utils} from '@natlibfi/melinda-commons';
+import * as config from './config';
+import startApp from './app';
+import {logError} from '@natlibfi/melinda-rest-api-commons';
+
+run();
+
+async function run() {
+  const {handleInterrupt} = Utils;
+  let server; // eslint-disable-line functional/no-let
```

Comment:
See how this is done in [here](https://github.com/NatLibFi/aleph-own-auth-api/blob/next/src/index.js).